binding.cc

```diff
@@ -83,7 +83,7 @@ void Execute(napi_env env, void* data) {
   uv_mutex_unlock(&(self->_mutex));
 
   std::string json(tr_variantToStr(&(self->response), TR_VARIANT_FMT_JSON));
-  self->response_str = new char[json.size()];
+  self->response_str = new char[json.size() + 1];
```

Comment:
WDYT about adding a little comment to explain the caveat?

Reply:
You r right, I added one, thanks :+1: